rebuild_protos.sh

```diff
@@ -0,0 +1,37 @@
+poetry shell
+
+# Clean existing protos
+proto_dir=./proto
+rm -r "$proto_dir"
+
+# Copy proto from substrait repo
+cp -fr ../substrait/proto ./
+
+# Changes permissions
+find "$proto_dir" -type d -exec chmod u+rwx {} +
+find "$proto_dir" -type f -exec chmod u+rw {} +
+
+# Clean compiled Python proto definitions
+rm -rf ./ibis_substrait/proto
+
+# Map the namespaces for the protos
+python proto_prefix.py "$proto_dir"/tmp substrait.ibis "$proto_dir"/substrait
+mv "$proto_dir"/substrait substrait.bak
+mv "$proto_dir"/tmp/substrait "$proto_dir"
+
+# Bring in the proto
+curl \
```

Comment:
I think to upstream this we would want the extension_rels.proto to land in the upstream source tree so we aren't dependent on a particular fork to pull this down.

Reply:
I'm also curious (not urgent) whether we can use the namespacing script to avoid the need for the `sed` commands below

Reply:
As in, [this PR](https://github.com/substrait-io/substrait/pull/331)?

Or is there some other PR?

Reply:
Ahh, I meant more that if it were living in `arrow` that we might pull it down from `apache/arrow`, but if it gets upstreamed into `substrait` that is even better.

Reply:
Got it. So perhaps long-term it will exist in Substrait, but that in the near term it might live as an extension in Arrow?

Reply:
Right.  And in the near-term it may not make sense to have those proto definitions in ibis-substrait at all, they may belong in a standalone package.